utest/jvm/src/test/scala-2.12/test/utest/FormatterTests.scala

```diff
@@ -9,7 +9,10 @@ import concurrent.ExecutionContext.Implicits.global
 object FormatterTests extends utest.TestSuite {
   def trim(s: String): String = {
     // Predef.augmentString = work around scala/bug#11125
-    Predef.augmentString(s.trim).lines.map(_.reverse.dropWhile(_ == ' ').reverse).mkString("\n").replaceAll(" \\d+ms", "")
+    Predef.augmentString(s.trim).lines.map(_.reverse.dropWhile(_ == ' ').reverse)
+      .mkString("\n")
+      .replaceAll(" \\d+ms", "")
+      .replaceAll(":\\d+\\)", ":\\)")
```

Comment:
the line numbers in scala-library changed in 2.12.8 compared to 2.12.6 so now we strip them out from stack traces